src/test/java/com/microsoft/azure/kusto/kafka/connect/sink/E2ETest.java

```diff
@@ -1,75 +1,81 @@
-import com.microsoft.azure.kusto.ingest.IngestClient;
-import com.microsoft.azure.kusto.ingest.IngestClientFactory;
 import com.microsoft.azure.kusto.ingest.IngestionMapping;
 import com.microsoft.azure.kusto.ingest.IngestionProperties;
 
-import java.io.FileInputStream;
 import java.io.IOException;
 import java.net.URISyntaxException;
+import java.nio.file.Files;
 import java.nio.file.Paths;
 import java.util.ArrayList;
 import java.util.Collections;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
-import java.util.Properties;
 import java.util.UUID;
-import java.util.logging.Logger;
 
 @Disabled("We don't want these tests running as part of the build or CI. Comment this line to test manually.")
 public class E2ETest {
     private static final String testPrefix = "tmpKafkaE2ETest";
-    private static final String appId = System.getProperty("appId");
-    private static final String appKey = System.getProperty("appKey");
-    private static final String authority = System.getProperty("authority");
-    private static final String cluster = System.getProperty("cluster");
-    private static final String database = System.getProperty("database");
-    private static final String tableBaseName = System.getProperty("table", testPrefix + UUID.randomUUID().toString().replace('-', '_'));
+    private static final Logger log = LoggerFactory.getLogger(FileWriter.class);
+    private static String appId;
+    private static String appKey;
+    private static String authority;
+    private static String cluster;
+    private static String database;
+    private static String tableBaseName;
     private static HttpProxyServer proxy;
     private final String basePath = Paths.get("src/test/resources/", "testE2E").toString();
-    private final Logger log = Logger.getLogger(this.getClass().getName());
-    private boolean isDlqEnabled;
-    private String dlqTopicName;
-    private static Producer<byte[], byte[]> kafkaProducer;
 
     @BeforeAll
     public static void beforeAll() {
-        Properties properties = new Properties();
-        properties.put("bootstrap.servers", "localhost:9000");
-        properties.put("key.serializer", "org.apache.kafka.common.serialization.ByteArraySerializer");
-        properties.put("value.serializer", "org.apache.kafka.common.serialization.ByteArraySerializer");
-        kafkaProducer = new KafkaProducer<>(properties);
+        /*
+         * Some scanners report global declarations as not handling exceptions and also not santizing names. Extracting these to beforeAll and then sanitizing
+         * the names should fix this.
+         */
+        appId = getProperty("appId", null, false); // not used in file paths for temp files
+        appKey = getProperty("appKey", null, false); // not used in file paths for temp files
+        authority = getProperty("authority", null, false); // not used in file paths for temp files

```

Comment:
what does this comment mean? also - why repeat it

Reply:
@ohadbitt  fixed this